File: SecondYear/jeezzzz_AjeeshRawal_2226cseai1021_2/Week_1/pattern8.java
class Solution {

    void printTriangle(int n) {
        int i,j,k,space=16;
        for(i=1;i<=n;i++)
        {
            for(j=1;j<=i;j++)
            {
                System.out.print(j+" ");
            }
            for(k=1;k<=space;k++)
            {
                System.out.print(" ");
            }
            space=space-4;
            for(j=i;j>=1;j--)
            {
                System.out.print(j+" ");
            }
            System.out.println();
        }
    }
}

File: SecondYear/swarnkarvikash_VikashKumarSwarnkar_2226csit1034_2/Week_1/Pattern8.cpp
class Solution {
  public:
    void printTriangle(int n) {
        // code here
         int space=(4*(n-1))+1;
       for(int i=1;i<=n;i++)
       {
           for(int j=1;j<=i;j++)
           {
               cout<<j<<" ";
           }
           for(int j=1;j<space;j++)
           {
               cout<<" ";
           }
           for(int j=i;j>=1;j--)
           {
               cout<<j<<" ";
           }
           cout<<endl;
           space-=4;
       }
    
    }
};